tests/PHPUnit/System/expected/test_BackwardsCompatibility1XTest_flat__Actions.getPageUrls_range.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <result>
 	<row>
-		<label>index</label>
+		<label></label>
```

Comment:
here it's expected to be `/` ie. the root URL

Reply:
In flattened url reports currently all leading slashes are removed. Should all other labels have leading slashes as well? Otherwise it would be inconsistent

Reply:
:+1:  sounds good that all pages have leading slashes